viadot/tasks/sap_rfc.py

```diff
@@ -52,12 +59,16 @@ def __init__(
             512 characters. However, we observed SAP raising an exception even on a slightly lower number
             of characters, so we add a safety margin. Defaults to 400.
             credentials (dict, optional): The credentials to use to authenticate with SAP.
+            saprfc_credentials_key (str, optional): Azure KV secret. Defaults to "SAP".
+            env (str, optional): SAP environment. Defaults to "PROD".
             By default, they're taken from the local viadot config.
```

Comment:
You can modify the docstring because it is a key for the credentials - it can be taken from Azure Key Vault or local config. It depends on what is possible.

Please change for the source and flow as well.